BasicController.java

```diff
@@ -36,7 +36,7 @@ public ApiResult multiply(@RequestParam BigDecimal multiplier, @RequestParam Big
    @GetMapping("/divide")
    public ApiResult divide(@RequestParam BigDecimal dividend, @RequestParam BigDecimal divisor) {
       if (divisor.equals(BigDecimal.ZERO)) {
```

Comment:
The divide method should return an `ApiResult` object, but the modified code returns a string directly when the divisor is zero, which is not consistent with the expected return type.